other/materials_designer/slab_passivation.ipynb

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
<p>Can we create a helper image for this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/170/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>